src/components /Pokemon.js

```diff
@@ -0,0 +1,46 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import './Pokemon.css';
+
+function Pokemon({endpoint}) {
+
+    const [pokemonData, setPokemonData] = useState({});
```

Comment:
Moet deze natuurlijk ook wel een default waarde van `undefined` hebben
```suggestion
    const [pokemonData, setPokemonData] = useState();
```